src/sample/courseReg.fxml

```diff
@@ -152,7 +152,11 @@
                         <Font size="18.0" />
                      </font>
                   </Text>
+<<<<<<< HEAD
```

Comment:
why got this HEAD thing?

Reply:
The <<<<<< HEAD and >>>>>>> thingy is actually the conflicts. But now idk which one to keep and remove le becuase I am not the one who edit this part. I afraid later I remove the wrong part and keep the wrong part.
====== is actually the divider between the <<<<<< and >>>>>>